ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -174,15 +175,16 @@ func (r *RayClusterReconciler) Reconcile(ctx context.Context, request ctrl.Reque
 
 	// No match found
 	if errors.IsNotFound(err) {
-		r.Log.Info("Read request instance not found error!", "name", request.NamespacedName)
+		logger.Info("Read request instance not found error!", "name", request.NamespacedName)
```

Comment:
We should remove the redundant parameters (`"name", request.NamespacedName`) from existing log messages, since these will be added through: `r.Log.WithValues("RayCluster", request.NamespacedName)`